### PR TITLE
MRDC-722

### PR DESCRIPTION
**Pull request by clample**
Client changes for multi DC Appium Suites

### CODE REVIEW

**Comment by clample** [comment]
TODO: update the "updateSuite" method for the new v2 Appium resource

File: src/main/java/org/testobject/rest/api/appium/common/data/DataCenterSuite.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
+package org.testobject.rest.api.appium.common.data;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.net.URL;
+import java.util.Set;
+
+public class DataCenterSuite {
```
Comment (umutuzgur) [inline src/main/java/org/testobject/rest/api/appium/common/data/DataCenterSuite.java]: Please always put JsonIgnore(ignoreUnknown) . It is a good idea to always put that into our public APIs

File: pom.xml [hunk 2]
```diff
@@ -3,7 +3,7 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>org.testobject</groupId>
 	<artifactId>testobject-java-api</artifactId>
-	<version>0.0.26</version>
+	<version>0.0.27</version>
```
Comment (umutuzgur) [inline pom.xml]: Since we are breaking the api, we should increase the minor version. So maybe 0.1.0 makes more sense

Comment (clample) [inline pom.xml]: Makes sense, I agree

File: src/main/java/org/testobject/rest/api/appium/common/data/DataCenterSuite.java (hunk 1 above)
Comment (clample) [inline src/main/java/org/testobject/rest/api/appium/common/data/DataCenterSuite.java]: Got it, makes sense. I'll do that

**Comment by clample** [comment]
Working on staging: https://app.staging.testobject.org/#/foobar/calculator/appium/4426/reports/2210

**Pull request merged**